Formula/minibox.rb

```diff
@@ -1,28 +1,25 @@
 # GENERATED FROM TEMPLATE. DO NOT EDIT!
 class  Minibox < Formula
-  desc "Minibox CLI"
+  desc "CLI"
```

Comment:
Thoughts on just deleting the minibox formula? It's not used or needed.

Reply:
Yeah, no reason to keep it, will remove.